gravitee-apim-console-webui/src/management/api/api-navigation/api-navigation-tabs/api-navigation-tabs.component.html

```diff
@@ -18,13 +18,22 @@
 <nav mat-tab-nav-bar class="api-navigation-tabs">
   <ng-container *ngFor="let menuItem of this.tabMenuItems">
     <a
-      *ngIf="menuItem.routerLink !== 'DISABLED'"
+      *ngIf="menuItem.routerLink !== 'DISABLED' && !!menuItem.routerLinkActiveOptions"
+      mat-tab-link
+      routerLinkActive
+      #rla="routerLinkActive"
+      [routerLink]="menuItem.routerLink"
+      [active]="rla.isActive"
+      [routerLinkActiveOptions]="menuItem?.routerLinkActiveOptions"
```

Comment:
maybe `menuItem?.routerLinkActiveOptions ?? {}` to avoid duplication ? not sure if it's work

Reply:
I tested it and it's not working but good catch I have change it too `menuItem?.routerLinkActiveOptions ?? { exact: false }"`